src/Microsoft.Identity.Web/MicrosoftIdentityCircuitHandler.cs

```diff
@@ -75,7 +77,9 @@ public ClaimsPrincipal User
             get
             {
                 return _user ??
+#pragma warning disable CS8602 // Dereference of a possibly null reference. HttpContext will not be null in this case.
                     (!IsBlazorServer ? _httpContextAccessor.HttpContext.User :
```

Comment:
Instead of pragmas, I would use a [! null-forgiving operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-forgiving) here.
Something like `_httpContextAccessor!.HttpContext!.User` probably.

Reply:
I know, i just have to add them everywhere, even in creating the request...and then there more and more and more. so it's not really necessary in this case, so that's why i went w/the pragmas.